modules/fscloud/README.md

```diff
@@ -0,0 +1,68 @@
+# Profile for IBM Cloud Framework for Financial Services
+
+This code is a version of the [parent root module](../../) that includes a default configuration that complies with the relevant controls from the [IBM Cloud Framework for Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about). See the [Example for IBM Cloud Framework for Financial Services](/examples/fscloud/) for logic that uses this module.
+
+:exclamation: **Exception:** The Databases for Elasticsearch service is not yet Financial services validated. Therefore, the infrastructure that is deployed by this module is also not validated with the Framework for Financial Services. For more information, see the list of [Financial Services Validated services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-vpc-architecture-about#financial-services-validated-services).
+
+The default values in this profile were scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) for compliance with the IBM Cloud Framework for Financial Services profile that is specified by the IBM Security and Compliance Center. The scan passed for all applicable goals with one exception:
+
+> rule-beb7b289-706b-4dc0-b01d-b1d15d4331e3: Check whether Databases for Elasticsearch network access is restricted to a specific IP range
```

Comment:
This is a MongoDB rule ID. I don't see any rule for elasticsearch added in cra-tf-validate-ignore-rules.json so its means there must not be any failing CRA rules for Elasticsearch

Reply:
hmmm it seems there is a rule for Elasticsearch: `rule-b5c06228-3f38-4d98-837f-2fe10d6ff9d5 - Check whether Databases for Elasticsearch network access is restricted to a specific IP range` - so why does CRA not catch it?

Reply:
@Aashiq-J looks like you can remove this all together - CRA must not longer check for it as it does some checks for CBR now. So make same change as I did in https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/pull/353